### DISCUSSION TITLE
Having trouble adding an extension

### DISCUSSION THREAD

**Discussion by dweng0** [quarto-dev/quarto-cli#6423] [Q&A] [answered]
### Description

I have created an extension that is available [here](https://github.com/clearmatics/qreacto)

However, when i go to add it to an existing project using:

``` bash
 quarto add clearmatics/qreacto
```

I get the following error

``` bash
Extension not found in local or remote sources
```

The project is open and public, any ideas what might be causing this?

**Comment by dragonstyle** [comment]
I'm not 100% sure, but I'm guessing the issue is that the folder name for you extension (in your repo) is `_extensions/reacto` . It should be `_extensions/qreacto`.

**Comment by mcanouil** [comment, marked as answer]
It appears Quarto uses the `main` branch by default, which is the default on new GitHub repository (since the 1st of October, 2020).
If you rename your branch to `main` it should work or use `quarto add clearmatics/qreacto@master`.

PS: `quarto create-project` is deprecated, use `quarto create project` in your readme.

**Reply by dweng0** [reply]
It looks like this was the issue , thank you

**Comment by dweng0** [comment]
thank you will update the repo and get back to you
